assets/src/application/ui/layout/descriptionList/types.ts

```diff
@@ -5,5 +5,5 @@ export interface DescriptionListItemProps {
 }
 
 export interface DescriptionListProps {
-	dataSource: JSX.Element[];
+	dataSource: DescriptionListItemProps[];
```

Comment:
@tn3rb please note that in order to preserve this type @manzoorwanijk gave his :brain: and some changes has been made